run

```diff
@@ -17,4 +17,4 @@ EXE_PATH=$(find $SCRIPT_PATH/target/java-compiler-*-with-dependencies.jar) || ex
 
 # execute the compiler with the provided arguments and return the exit code
 java -jar "$EXE_PATH" "$@"
-exit $?
+exit $?
```

Comment:
Is this part of fixing the lexing of integer literals?

Reply:
No, it's an unrelated commit I hadn't noticed.